### ISSUE TITLE
Homeassistant requires version at the latest on 2021.6 release

### ISSUE THREAD

**Issue by watanen1** [jomwells/ambihue#11]
Homeassistant won't open this custom_component if the version is not present on the 2021.6 release. Please add version, I really like to custom component :)

**Comment by chillkiller** [comment]
@watanen1 open \\HOMEASSISTANT\config\custom_components\philips_ambilight(+hue)\manifest.json with an editor and insert:

{
  "domain": "philips_ambilight+hue",
  "name": "Philips Ambilight+Hue",
  "version": "42ae8b8",
  "documentation": "https://github.com/jomwells/ambihue",
  "dependencies": [],
  "codeowners": ["@jomwells"],
  "requirements": []
}

this development seems to be dead by jomwells
Also since last TV-Update you need to "re-pair" the tv with the components (python script for username and password)

Also there is hopefully soon an alternitive: https://github.com/home-assistant/core/pull/44867

**Comment by Mr-Groch** [comment]
I've made fork with this and some other changes (no need to setup node id - it uses native Phlips Ambilight+Hue API)

https://github.com/Mr-Groch/ambihue

**Comment by chillkiller** [comment]
> 
> 
> I've made fork with this and some other changes (no need to setup node id - it uses native Phlips Ambilight+Hue API)
> 
> https://github.com/Mr-Groch/ambihue

Do it work by yours??

I've tried, but it doesnt work. the entity exists (it should if it is set in config.yaml), but not reachable or switchable

**Comment by Mr-Groch** [comment]
> > I've made fork with this and some other changes (no need to setup node id - it uses native Phlips Ambilight+Hue API)
> > https://github.com/Mr-Groch/ambihue
> 
> Do it work by yours??
> 
> I've tried, but it doesnt work. the entity exists (it should if it is set in config.yaml), but not reachable or switchable

Yes, it is working for me... But there is no logic to detect if Philips TV uses older http or newer https connection... I have used older mode in

https://github.com/Mr-Groch/ambihue/blob/8aa2b95044a46d2c4ebb258f493ef0629cb3a64d/custom_components/philips_ambilight%2Bhue/switch.py#L18

In this line change:

`BASE_URL = 'http://{0}:1925/6/{1}'`

to:

`BASE_URL = 'https://{0}:1926/6/{1}'`

And it should work with newer TVs... Original (this) repo has https as default....

**Comment by Mr-Groch** [comment]
I've updated my fork to 0.0.4, so it's not needed to edit `.py` file anymore - `secured_transport` is configurable:

- Added new configurable options `secured_transport` (default `true`) and `api_version` (default `6`). If `secured_transport` is set to `false` (older and non Android TV's) then `username:` and `password:` is not required

**Comment by chillkiller** [comment]
@Mr-Groch 
Could it be that you are quite familiar with jointspace / Philips tv? If so, maybe the HA integration from https://github.com/home-assistant/core/pull/44867 would also be very interesting for you. I ask because your colleague @elupus seems to have stalled a bit at his work and maybe could use some help ...

Anyway, I'll try the new release tonight and give you feedback, but first of all thank you in advance ;-)

**Comment by chillkiller** [comment]
@Mr-Groch 
> 
> 
> I've updated my fork to 0.0.4, so it's not needed to edit `.py` file anymore - `secured_transport` is configurable:
> 
>     * Added new configurable options `secured_transport` (default `true`) and `api_version` (default `6`). If `secured_transport` is set to `false` (older and non Android TV's) then `username:` and `password:` is not required

Thanks alot, that was it...

my older philips tv (1925/6) died early last year an I bought a new one (1926/6), but i forgot that i changed switch.py after the tv update with the new url... yeah

It works like a charm ;)
